tests/python/test_cli.py

```diff
@@ -241,7 +241,14 @@ def test_cli_debug():
 
 
 def test_cli_run():
-    with patch_sys_argv_helper(["ti", "run", "test_task", "arg1",
+    with patch_sys_argv_helper(["ti", "run", "a.py"]) as custom_argv:
```

Comment:
What's `a.py`?

Reply:
Ask @rexwangcc, I copied from `test_cli_debug`.

Reply:
Oh, this function looks exactly the same as `test_cli_debug`. I'm not sure about `test_mode=True` in the next line. Everything other than this function LGTM.

Reply:
Thanks for being late here, the `a.py` is just a bogus name for a random script (maybe `foo.py` is more appropriate here) for testing. 

`test_mode=True` is a short circuit flag to tell the CLI exit early so tests won't trigger any real side-effects (see https://github.com/taichi-dev/taichi/pull/1391/files#diff-54e9c326d68839f8cc6d23471881df32R889-R901). This was to make the testing easier, not sure if it's optimal tho.